BootloaderCorePkg/Tools/IfwiUtility.py

```diff
@@ -48,6 +48,23 @@ def set_values(self, _address, _size, _version, _type, _checksum):
         self.type     = _type
         self.checksum = _checksum
 
+class FIT_ENTRY_V200(Structure):
```

Comment:
This is same as original FIT_ENTRY. Redefinition is not needed.
We have to update the base FIT_ENTRY definition as per the latest entries 
https://github.com/andreiw/lampone-edk2-platforms/blob/0d5a9b9e01b1fd5084276b60a49980bdfb00cc94/Silicon/Intel/Tools/FitGen/FitGen.c#L24 

Our base structure needs only filed updates.